packages/manager/modules/exchange/src/translations/Messages_fr_FR.json

```
@@ -1249,23 +1261,24 @@
   "exchange_ACTION_configure_help_minPasswordAge_title": "Durée de vie minimale des mots de passe",
-  "exchange_ACTION_configure_help_minPasswordAge_text": "Ce paramètre détermine la période de temps (en jours) durant laquelle un mot de passe doit être utilisé avant que l'utilisateur puisse le modifier. Vous pouvez définir une valeur comprise entre 1 et 90 jours ou bien vous pouvez autoriser immédiatement des modifications en définissant ce nombre de jours sur la valeur 0.",
+  "exchange_ACTION_configure_help_minPasswordAge_text": "<p>Ce paramètre détermine la période de temps (en jours) durant laquelle un mot de passe doit être utilisé avant que l'utilisateur puisse le modifier.</p><p> Vous pouvez définir une valeur comprise entre 1 et 90 jours ou bien vous pouvez autoriser immédiatement des modifications en définissant ce nombre de jours sur la valeur 0.</p>",
   "exchange_ACTION_configure_help_maxPasswordAge_title": "Durée de vie maximale des mots de passe",
-  "exchange_ACTION_configure_help_maxPasswordAge_text": "Ce paramètre de sécurité détermine la période de temps (en jours) durant laquelle un mot de passe peut être utilisé avant que le système ne demande à l'utilisateur de le modifier. Vous pouvez définir ce paramètre pour que les mots de passe expirent après un nombre de jours compris entre 1 et 90 ou vous pouvez spécifier que les mots de passe ne doivent jamais expirer en définissant le nombre de jours sur la valeur 0. Si la durée de vie maximale du mot de passe est comprise entre 1 et 90 jours, la durée de vie minimale des mots de passe doit être inférieure à la durée de vie maximale de celui-ci. Si la durée de vie maximale du mot de passe est définie sur la valeur 0, la durée de vie minimale peut être définie sur n'importe quelle valeur comprise entre 0 et 90 jours.",
+  "exchange_ACTION_configure_help_maxPasswordAge_text": "<p>Ce paramètre de sécurité détermine la période de temps (en jours) durant laquelle un mot de passe peut être utilisé avant que le système ne demande à l'utilisateur de le modifier.</p><p>Vous pouvez définir ce paramètre pour que les mots de passe expirent après un nombre de jours compris entre 1 et 90 ou vous pouvez spécifier que les mots de passe ne doivent jamais expirer en définissant le nombre de jours sur la valeur 0.</p><p>Si la durée de vie maximale du mot de passe est comprise entre 1 et 90 jours, la durée de vie minimale des mots de passe doit être inférieure à la durée de vie maximale de celui-ci.</p><p> Si la durée de vie maximale du mot de passe est définie sur la valeur 0, la durée de vie minimale peut être définie sur n'importe quelle valeur comprise entre 0 et 90 jours.</p>",
```

Extra white space (at the beginning of the latest `<p>` element).

```suggestion
  "exchange_ACTION_configure_help_maxPasswordAge_text": "<p>Ce paramètre de sécurité détermine la période de temps (en jours) durant laquelle un mot de passe peut être utilisé avant que le système ne demande à l'utilisateur de le modifier.</p><p>Vous pouvez définir ce paramètre pour que les mots de passe expirent après un nombre de jours compris entre 1 et 90 ou vous pouvez spécifier que les mots de passe ne doivent jamais expirer en définissant le nombre de jours sur la valeur 0.</p><p>Si la durée de vie maximale du mot de passe est comprise entre 1 et 90 jours, la durée de vie minimale des mots de passe doit être inférieure à la durée de vie maximale de celui-ci.</p><p>Si la durée de vie maximale du mot de passe est définie sur la valeur 0, la durée de vie minimale peut être définie sur n'importe quelle valeur comprise entre 0 et 90 jours.</p>",
```